docs/guides/fedora/_include/live-environment.rst

```diff
@@ -16,12 +16,17 @@ Install updated ZFS packages
 .. code-block::
 
    rpm -e --nodeps zfs-fuse
-   dnf config-manager --disable updates
-   dnf install -y https://zfsonlinux.org/fedora/zfs-release-2-5$(rpm --eval "%{dist}").noarch.rpm
+   dnf config-manager setopt updates.enabled=0
+   dnf install -y https://zfsonlinux.org/fedora/zfs-release-3-0$(rpm --eval "%{dist}").noarch.rpm
    dnf install -y https://dl.fedoraproject.org/pub/fedora/linux/releases/${VERSION_ID}/Everything/x86_64/os/Packages/k/kernel-devel-$(uname -r).rpm
    dnf install -y zfs gdisk
    modprobe zfs
 
+.. note::
+
+   - The ``updates`` repository is temporarily disabled to ensure that the correct kernel-headers package can be located and installed.
+   - Refer to the list of available zfs-release RPMs `here <https://github.com/zfsonlinux/zfsonlinux.github.com/tree/master/fedora>`_.
```

Comment:
If you want to note this link, don't make "here" the label. Use "list of available zfs-release RPMs" instead.